frontend/src/components/layout/Header/Header.tsx

```diff
@@ -97,12 +97,12 @@ const Header = () => {
           <h1 className="shortAppName">PIMS</h1>
         </Nav.Item>
       </Nav>
+      {!history.location.pathname.includes('/login') && <UserProfile />}
```

Comment:
I think this is potentially fragile if we end up with more then one page where the user is not logged in. Should be able to base this off of the keycloak status.

Reply:
Will adjust